local-test-operator.yml

```diff
@@ -150,13 +151,31 @@
             openshift_images_before: " {{ os_registry_is_result_before.stdout_lines }} + {{ os_pod_is_result_before.stdout_lines }}"
           when: not run_upstream|bool
 
+        - name: "Set the deployment start time"
+          shell: "date -u '+%Y-%m-%dT%H:%M:%SZ'"
```

Comment:
Could you explain why we cannot use epoch here?

Reply:
Do you mean the "seconds since the epoch" format or the literal epoch time?

In the first case, the `oc get events` command I linked in my comment above uses that format for the date.  Getting the date in the same format makes it easier to compare.  The other option is to convert the date from the `oc get events` command to seconds since the epoch.  That requires extra steps 

In the second case, that would give us all images pulled by all namespaces going back as far as the cluster keeps events logs.  The operator bundles will be running on a "clean" cluster, but there could still be images pulled for the cluster deployment that we don't want to drag into the test

Reply:
Such explanation should be in the code as a comment.
Just in case something will change in the future so we will know why and what need to be changed.

Reply:
I still cannot see any explanation in the code.
@14rcole could you explain why did you resolved discussion?

Reply:
ehm, it was explained in the roles/collect_modified_images/tasks/main.yml